src/telemetry/telemetryService.ts

```diff
@@ -193,14 +194,6 @@ export enum TelemetryEventTypes {
   PARTICIPANT_RESPONSE_GENERATED = 'Participant Response Generated',
 }
 
-export enum ParticipantErrorTypes {
```

Comment:
I figured it'd be better to get the error types out of the scope of telemetry since it might be useful to handle specific errors.